src/main/java/io/appform/jsonrules/utils/ComparisonUtils.java

```diff
@@ -86,7 +88,12 @@ static int compare(JsonNode evaluatedNode, String value) {
     public static boolean compareForEquality(ExpressionEvaluationContext context, JsonNode evaluatedNode,
             Object value) {
         final boolean nodeMissingOrNullCheck = isNodeMissingOrNull(evaluatedNode);
-        JsonNode jsonNode = mapper.valueToTree(JsonPath.read(context.getNode().toString(), String.valueOf(value)));
+        JsonNode jsonNode = MissingNode.getInstance();
+        try {
+            jsonNode = mapper.valueToTree(JsonPath.read(context.getNode().toString(), String.valueOf(value)));
+        } catch (PathNotFoundException e) {
+            // consume silently; indicates path @value doesn't exist.
```

Comment:
Is there any way to test for this? Doing this through exceptions will be really non-performant.

Reply:
Let me add a couple of test cases to ease this.
Identifying non-existence of a path through exception is better as we can open it up for a user to take a call on the eventual result. Also, PNFE comes when path doesn't exist shouldn't hinder performance. 

Further, jayway does give a way to muzzle down exceptions during path evaluations through SUPPRESS_EXCEPTION option. But, it comes at the cost of loosing out on the aforementioned flexibility of user deciding on the outcome.